src/components/generics/vacation-test-forms/myVacationMockData.json

```diff
@@ -0,0 +1,62 @@
+[
```

Comment:
I'm not seeing any tests, is this just something you are using locally, in which case does it need to be committed? Perhaps you can either delete it when you are finished, or not commit it, or add it to the .gitignore but I don't see any reason for this to be here. Let me know if I'm mistaken